crypto/bio/bss_conn.c

```diff
@@ -271,12 +271,14 @@ BIO_CONNECT *BIO_CONNECT_new(void)
         return (NULL);
     ret->state = BIO_CONN_S_BEFORE;
     ret->connect_family = BIO_FAMILY_IPANY;
+/*
     ret->param_hostname = NULL;
     ret->param_service = NULL;
     ret->info_callback = NULL;
     ret->connect_mode = 0;
     ret->addr_first = NULL;
     ret->addr_iter = NULL;
+*/
```

Comment:
There is no point in commenting out those lines, either remove them or let them be.